Wox.Test/FuzzyMatcherTest.cs

```diff
@@ -130,6 +133,9 @@ public void WhenGivenStringsForCalScoreMethodThenShouldReturnCurrentScoring(stri
 
             var results = new List<Result>();
 
+            Alphabet.Initialize(false);
+            StringMatcher.UserSettingSearchPrecision = StringMatcher.SearchPrecisionScore.None.ToString();
```

Comment:
Are we sure we want to use none? that basically says no precision filtering is apply on the result, I would say regular is preferred